framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -101,9 +101,9 @@
  * two maps passed as arguments.
  *
  * <p>The main entry point is {@link AnnotationFileParser#parse(String, InputStream,
- * AnnotatedTypeFactory, ProcessingEnvironment, StubAnnotations)}, which side-effects its last two
- * arguments. It operates in two steps. First, it calls the Stub Parser to parse a stub file. Then,
- * it walks the Stub Parser's AST to create/collect types and declaration annotations.
+ * AnnotatedTypeFactory, ProcessingEnvironment, AnnotationFileAnnotations)}, which side-effects its
+ * last two arguments. It operates in two steps. First, it calls the Stub Parser to parse a stub
```

Comment:
I think now it only side-effects the last argument rather than the last two. Also, on this comment can also change "stub" to "annotation file".